crates/tests/core/tests/unknown.rs

```diff
@@ -0,0 +1,54 @@
+#![allow(non_snake_case)]
+
+use windows::core::*;
+
+#[interface("f7ea748b-8121-41c1-aaee-406ba6f148a9")]
+unsafe trait ITest: IUnknown {
+    unsafe fn Test(&self) -> u32;
+}
+
+#[implement(ITest)]
+struct Test {
+    drop: *mut u32,
```

Comment:
Just out of curiousity. Can this be made safe or does the `implement` macro not handle lifetimes?

```rust
#[implement(ITest)]
struct Test<'a> {
    drop: &mut u32,
}
```

Reply:
Yep it doesn't handle lifetimes. #1885